providers/gcore/convert.go

```diff
@@ -45,6 +45,11 @@ func nativeToRecords(n gcoreRRSetExtended, zoneName string) ([]*models.RecordCon
 				return nil, fmt.Errorf("unparsable record received from G-Core: %w", err)
 			}
 
+		case "SCVB": // GCore mistypes "SVCB" as "SCVB"
```

Comment:
Maybe support both in case they correct their typo?

```suggestion
		case "SCVB", "SVCB": // GCore mistypes "SVCB" as "SCVB"
```

Reply:
If GCore fixes their typo, it will be handled by the default case, which has the exact same logic as the `SCVB` case, including the type name (both passing `SVCB`).

Reply:
Oh!  Of course!  Sorry, I didn't look too closely.